Polyfills/Canvas/Source/Font.h

```diff
@@ -13,18 +13,20 @@ namespace Babylon::Polyfills::Internal
 
     struct Font
     {
+    public:
+        operator std::string() const;
+        static std::optional<Font> Parse(const std::string& fontString);
+
+        float GetSize() const { return size; }
+        const std::string& GetFamiliy() const { return family; }
+
     private:
         static constexpr int NORMAL_WEIGHT = 400;
         static constexpr int BOLD_WEIGHT = 700;
 
-    public:
-        FontStyle Style = FontStyle::Normal;
-        int Weight = NORMAL_WEIGHT;
-        float Size = 10;
-        std::string Family = "sans-serif";
-
-        operator std::string() const;
-
-        static std::optional<Font> Parse(const std::string& fontString);
+        FontStyle style{FontStyle::Normal};
+        int weight{NORMAL_WEIGHT};
+        float size{10};
+        std::string family{"sans-serif"};
```

Comment:
These variables should have `m_` prefix.

Reply:
Even public? I commented on them being public, so if we switch and have get functions, then it seems like they should be `m_`

Reply:
I think this PR made all of these variables private?

Reply:
fixed